src/bio2bel/sources/intact.py

```diff
@@ -4,18 +4,18 @@
 
 Run with ``python -m bio2bel.sources.intact``
 
-`IntAct <https://www.ebi.ac.uk/intact/>`_ is a interaction database with information about interacting proteins,
+`IntACt <https://www.ebi.ac.uk/intact/>`_ is a interaction database with information about interacting proteins,
```

Comment:
still wrong. Should be `IntAct`

Maybe it doesn't seem like it, but this kind of attention to detail is very important